controllers/post.js

```diff
@@ -24,4 +24,42 @@ const newPost = async (req, res) => {
   }
 };
 
-export { newPost };
+export const getPost = async (req, res) => {
+  try {
+    const { postId } = req.params;
+
+    const post = await prisma.post.findUnique({
+      where: {
+        id: parseInt(postId),
+      },
+      include: {
+        user: {
+          select: {
+            id: true,
+            fullname: true,
+            profile_photo: true,
+          },
+        },
+        comment: {
```

Comment:
If possible, rename this field to `comments` since there may be more than one comment on a single post.